corehq/apps/change_feed/management/commands/create_kafka_topics.py

```diff
@@ -0,0 +1,21 @@
+from optparse import make_option
+from django.core.management import BaseCommand
+from corehq.apps.change_feed import topics
+from corehq.apps.change_feed.connection import get_kafka_client
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        create_kafka_topics()
+
+
+def create_kafka_topics():
+    client = get_kafka_client()
+    for topic in topics.ALL:
+        if client.has_metadata_for_topic(topic):
+            status = "already exists"
+        else:
+            client.ensure_topic_exists(topic, timeout=5)
```

Comment:
This is a much faster way to create topics than the bash script it replaced. For this to work, Kafka [must be configured with `auto.create.topics.enable = true`](https://github.com/dpkp/kafka-python/issues/438#issuecomment-127815720) (that's [the default value](https://kafka.apache.org/08/configuration.html)). I didn't change anything in our kafka test setup to make this work.